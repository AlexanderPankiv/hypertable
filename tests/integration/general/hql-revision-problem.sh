#!/usr/bin/env bash

HT_HOME=${INSTALL_DIR:-"$HOME/hypertable/current"}
SCRIPT_DIR=`dirname $0`

trap "$HT_HOME/bin/ht-stop-servers.sh" EXIT

$HT_HOME/bin/ht-start-test-servers.sh --clear --no-thriftbroker

cat $SCRIPT_DIR/hql-revision-problem.hql | $HT_HOME/bin/ht shell --test-mode >& hql-revision-problem.output

diff $SCRIPT_DIR/hql-revision-problem.golden hql-revision-problem.output
if [ $? -ne 0 ]; then
    echo "error: diff returned non-zero, exiting..."
    exit 1
fi

exit 0

