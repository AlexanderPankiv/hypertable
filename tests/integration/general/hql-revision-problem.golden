CREATE NAMESPACE revision_problem;
USE revision_problem;
DROP TABLE IF EXISTS www;
CREATE TABLE www (
size MAX_VERSIONS 1
);
INSERT INTO www VALUES
("2016-04-10 02:46:55.892228825", "row01", "size:content", "70772"),
("2016-04-11 02:43:21.605421596", "row01", "size:content", "70772");
SELECT size FROM www WHERE CELL="row01","size:content" DISPLAY_TIMESTAMPS;
2016-04-11 02:43:21.605421596	row01	size:content	70772
SELECT size FROM www WHERE CELL="row01","size:content" and TIMESTAMP < "2016-04-11 00:00:00" DISPLAY_TIMESTAMPS;
